Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure(2) do |config|
+    config.vm.box = "fedora/25-cloud-base"
+    config.vm.network "forwarded_port", guest: 9090, host: 9090
```

Comment:
should we pick a different port on the host, in case it also has cockpit running?